accessor/mysql_status.go

```diff
@@ -17,8 +17,16 @@ import (
 
 // MySQLClusterStatus defines the observed state of MySQLCluster
 type MySQLClusterStatus struct {
-	InstanceStatus []MySQLInstanceStatus
-	Latest         *int
+	InstanceStatus             []MySQLInstanceStatus
+	Latest                     *int
+	IntermediatePrimaryOptions *IntermediatePrimaryOptions
+}
+
+type IntermediatePrimaryOptions struct {
+	MasterHost     string `db:"MasterHost"`
+	MasterUser     string `db:"MasterUser"`
+	MasterPassword string `db:"MasterPassword"`
```

Comment:
In the future, I would like to be able to get these status through the API.
I don't welcome exposing a password.